docs/src/languages/php/composer-auth.md

```diff
@@ -27,33 +32,36 @@ List that repository in your `composer.json` file.
 }
 ```
 
-## Set a project variable
+## 2. Set up Composer authentication using a variable
 
-Set the Composer authentication by adding a project level variable called `env:COMPOSER_AUTH` as JSON and available only during build time.
+To allow Composer to successfully authenticate when accessing the declared private repository,
+set an `env:COMPOSER_AUTH` variable for your project.
 
-That can be done through the [Console](/administration/web/_index.md) or via the command line, like so:
+To do so, run the following command:
 
 ```bash
 platform variable:create --level project --name env:COMPOSER_AUTH \
   --json true --visible-runtime false --sensitive true --visible-build true \
   --value '{"http-basic": {"{{< variable "PRIVATE_REPOSITORY_URL" >}}": {"username": "{{< variable "USERNAME" >}}", "password": "{{< variable "PASSWORD" >}}"}}}'
 ```
 
-The `env:` prefix makes that variable appear as its own Unix environment variable available by Composer during the build process.
-The optional `--visible-runtime false` and `--visible-build true` flags mean the variable is only defined during the build hook,
-which offers slightly better security.
+The `env:` prefix means that the variable is exposed as its own Unix environment variable,
+available to Composer only during build time.
 
-Note: The authentication credentials may be cached in your project's build container.
-Make sure you clear the Composer cache after changing credentials by running the following [CLI command](../../administration/cli/_index.md):
+## 3. Clear your project's build cache
+
+For security reasons, make sure that the authentication credentials aren't cached in your project's build container.
+To do so, run the following command:
 
 ```bash
 platform project:clear-build-cache
 ```
 
 ## Private repository hosting
```

Comment:
:lady_beetle: This section is a little unclear to me. I think your changes make it more clear, but maybe adjusting the heading would help too. It's not immediately clear to me when the section would apply to me. Maybe if the heading were about what action was involved it would help?

Reply:
I reworked the section, hopefully things are clearer now!